src/client/app/components/ThreeDPillComponent.tsx

```diff
@@ -40,6 +41,19 @@ export default function ThreeDPillComponent() {
 		return { meterOrGroupID: groupID, isDisabled: isDisabled, meterOrGroup: MeterOrGroup.groups } as MeterOrGroupPill;
 	});
 
+	// when there is only one choice, it must be selected as a default (there is no other option)
+	useEffect(() => {
+		const combinedPillData = [...meterPillData, ...groupPillData];
+
+		if (combinedPillData.length === 1) {
+			const singlePill = combinedPillData[0];
+			dispatch(updateThreeDMeterOrGroupInfo({
+				meterOrGroupID: singlePill.meterOrGroupID,
+				meterOrGroup: singlePill.meterOrGroup
+			}));
+		}
+	}, [meterPillData, groupPillData, dispatch]);
```

Comment:
I'm unsure why this depends on dispatch. Removing that dependency did not impact the code when I tired it.